src/app/details/details-side-nav/details-side-nav.component.html

```diff
@@ -14,14 +14,25 @@
       <div class="mat-caption" mat-line>GMC No:</div>
       <div mat-line>{{ traineeDetails.gmcNumber }}</div>
     </mat-list-item>
-    <mat-list-item *ngIf="traineeDetails.programmeMembershipType">
+    <mat-list-item>
       <div class="mat-caption" mat-line>Programme membership type:</div>
-      <div mat-line>{{ traineeDetails.programmeMembershipType }}</div>
+      <div mat-line>
+        {{
+          traineeDetails.programmeMembershipType === null
+            ? "N/A"
+            : traineeDetails.programmeMembershipType
+        }}
+      </div>
     </mat-list-item>
     <mat-list-item *ngIf="traineeDetails.programmeName">
-      <div class="mat-caption" mat-line>Programme name:</div>
-      <div mat-line>{{ traineeDetails.programmeName }}</div>
+      <div class="mat-caption" mat-line>Current programme name:</div>
+      <div
+        mat-line
+        class="programme-names"
+        [innerHTML]="traineeDetails.programmeName | splitStringToHTML"
+      ></div>
```

Comment:
As Pepe spotted, we could still need current programme name label even if it's empty.

Reply:
Fixed